server/clearbit.js

```diff
@@ -30,31 +35,41 @@ export default class Clearbit {
     };
 
     if (api_key) {
-      this.client = new Client(api_key, this.metric, this.log);
+      this.client = new Client(api_key, this.metric, this.clearbitLogger);
     }
   }
 
-  log = (msg, data = "") => {
-    this.hull.logger.info(msg, data);
-  }
+  clearbitLogger = (msg, data = {}) => {
```

Comment:
do we need those helper methods? Can we user the `hull.logger` and `hull.asUser().logger` directly?